src/test/java/gameframework/motion/IntersectToolsTest.java

```diff
@@ -161,4 +161,15 @@ public void setSpeedVector(SpeedVector m) {
 		fail("No need to set the speed vector");
 	}
 
+	@Override
+	public boolean isMovable() {
```

Comment:
are you sure this is needed? I have the impression that the superclass already has exactly the same method.